### PR TITLE
fuel-core `--version` doesn't work with `v0.4.1`

### LINKED ISSUE #205: `fuel-core --version` doesn't work with `v0.4.1`
Used to work with `v0.3.2`.

Seems similar to https://github.com/FuelLabs/sway/issues/881

### PR DESCRIPTION
**Pull request by ControlCplusControlV**
Fixes #205 , mirror style and changes as much as possible of the commit which fixed this in Sway, but if any changes are needed just let me know and I can get that fixed!

Works with both `fuel-core --version` and `fuel-core -V`

### CODE REVIEW

**Comment by Voxelot** [comment]
There's also a dev cli in fuel-client that may need to be updated.

**Comment by ControlCplusControlV** [comment]
Latest commit should fix it, testing with both
`./target/debug/fuel-gql-cli -V`
`./target/debug/fuel-gql-cli --version`

returning
`fuel-gql-cli 0.4.1`

File: fuel-core/src/args.rs [hunk 1]
```diff
@@ -9,6 +9,8 @@ lazy_static::lazy_static! {
 }
 
 #[derive(Parser, Debug)]
+#[clap(name = "fuel-core", about = "Fuel client implementation", version)]
+
```
Comment (adlerjohn) [inline fuel-core/src/args.rs:13]: Not: remove newline here for style consistency

Comment (ControlCplusControlV) [reply fuel-core/src/args.rs]: Removed

**Review by adlerjohn — APPROVED** [review]
Ahhh I'm approoooving

**Review by Voxelot — APPROVED** [review]

**Pull request merged**